cvat-ui/src/components/projects-page/project-item.tsx

```diff
@@ -38,6 +39,26 @@ export default function ProjectItemComponent(props: Props): JSX.Element {
 
     const style: React.CSSProperties = {};
 
+    const NUMBER_OF_RAWS = 3;
+    useEffect(() => {
```

Comment:
What do you think, can it be helpful to extract this to a separate component or hook to use it for example in cloud storages view?

Reply:
Makes sense

Reply:
Added user hook useCardHeight